cloudinary-core/src/main/java/com/cloudinary/Radius.java

```diff
@@ -0,0 +1,78 @@
+package com.cloudinary;
+
+import java.io.Serializable;
+
+
+public class Radius implements Serializable {
+
+private Object radius;
+    String concat = "";
+
+    public Radius() {
+    }
+
+    public Radius(Object value) {
+        if(value==null || value=="")
+            throw new IllegalArgumentException("Must supply value for the radius");
+        else {
+            this.radius = value;
+            Transformation transformation = new Transformation();
+            transformation.radius(value);
+        }
+    }
+
+    public Radius(Object topLeftAndBottomRight, Object topRightAndBottomLeft) {
+        if(topLeftAndBottomRight==null || topRightAndBottomLeft==null)
+            throw new IllegalArgumentException("Must supply two values for the radius ");
+        else{
+            radius = topLeftAndBottomRight + ":" + topRightAndBottomLeft;
+            Transformation transformation = new Transformation();
+            transformation.radius(radius);
+        }
+    }
+
+    public Radius(Object topLeft, Object topRightAndBottomLeft, Object bottomRight) {
+        if(topLeft==null || topRightAndBottomLeft==null || bottomRight==null)
+            throw new IllegalArgumentException("Must supply three values for the radius ");
+        else {
+            radius = topLeft + ":" + topRightAndBottomLeft + ":" + bottomRight;
+            Transformation transformation = new Transformation();
+            transformation.radius(radius);
+        }
+    }
+
+    public Radius(Object topLeft, Object topRight, Object bottomRight, Object bottomLeft) {
+        if(topLeft==null || topRight==null || bottomRight==null || bottomLeft==null )
+            throw new IllegalArgumentException("Must supply four values for the radius ");
+        else{
+            radius = topLeft + ":" + topRight + ":" + bottomRight + ":" + bottomLeft;
+            Transformation transformation = new Transformation();
+            transformation.radius(radius);
+        }
+    }
+
+    public Radius(Object[] array){
+        if(array==null || array.length==0 || array.length>4)
+            throw new IllegalArgumentException("Array length cannot be 0 or more than 4");
+        else {
+            for (Object value : array) {
+                if(value==null)
+                    throw new IllegalArgumentException("array value cannot be null");
+                    concat = concat + ":" + value;
```

Comment:
@aditimadan-Cloudinary , since we get an Object, what will happen if the user sends unexpected types of object? Map for example. 
Also, are we handling expressions?

Reply:
We usually don't validate parameter type, we just `toString()` whatever is sent, for future proofing. This is not ideal but it's consistent with the current SDK implementation. If an expression is sent it's own `toString()` implementations handles the normalisation and templating so it should work - @aditimadan-Cloudinary  please add a test with an expression to be on the safe side.